saleor/graphql/webhook/subscription_types.py

```diff
@@ -790,14 +791,29 @@ class InvoiceBase(AbstractType):
         "saleor.graphql.invoice.types.Invoice",
         description="The invoice the event relates to.",
     )
+    order = graphene.Field(
+        "saleor.graphql.order.types.Order",
+        description="Order related to the invoice.",
```

Comment:
Missing ADDED_IN_310. (I think it is missing in all added fields)

Reply:
Great point, thanks! Done